### ISSUE TITLE
[Server Request] Soldier of Fortune 2: Double Helix (Gold Edition)

### ISSUE THREAD

**Issue by tkrn** [GameServerManagers/LinuxGSM#2210]
## What game server would you like adding?

Soldier of Fortune 2: Double Helix (Gold Edition)

## Can you link to any tutorials or guides?
 
The game is based upon the Quake 3 engine as it was licensed from id Software to Raven Software during 2002. Since then there has been a server side (only) modification called 1fx. Mod (http://1fxmod.org/start-linux/) which bring modern features/customizations which you would expect out of typical Quake 3 style dedicated server. There is no modifications for the client. There is a particular item I may need help addressing, it appears that sof2ded the binary for the dedicated server looks for a file explicitly in /usr/lib. Does LGSM handle privilege escalation for a sym link or copying of the libcxa.so.1 library? How should I handle this potentially? 

Also, please feel free to assign me this Server Request/Issue. 

## Is the server on Steam?

* [ ] Yes
* [X] No

The game is not on Steam by on GOG.com

## Is the server supported on Linux?

We only support Linux servers and do not support Wine.

* [X] Yes
* [ ] No

**Comment by dgibbs64** [comment]
I will add the server files to linxugsm repo soon

**Comment by tkrn** [comment]
Server files ready for upload. Grab them at http://tkrn.io/github/SOF2Gold-1.03.tar.bz2

I will have the server.cfg and mapcycle.txt added to to the Game-Server-Config added soon along with the rest of the install modifications.

**Comment by dgibbs64** [comment]
Uploaded the only change is I altered the name to lowercase  https://files.linuxgsm.com/SoldierOfFortune2

**Comment by tkrn** [comment]
See pull request to the development branch - https://github.com/GameServerManagers/LinuxGSM/pull/2359

**Comment by jobhh** [comment]
@tkrn I've installed a clean sof2 server vis LinuxGSM and tried to connect with a clean GOG sof2 installation. I got the error shown in the image below while the default server setting is "seta sv_allowDownload "1"".

Anyway, while investigating I noticed there is no 1fx folder after the installation.
I read in this issue that you mentioned the server side 1fx. Mod, was this included in the LinuxGSM installation?


![sof2_server_issue](https://user-images.githubusercontent.com/2331451/98388738-6db6c100-2053-11eb-83cd-b16a2901bcac.jpeg)

**Comment by dgibbs64** [comment]
@jobhh please raise this as a new issue. Thanks

**Comment by jobhh** [comment]
@dgibbs64 No problem. I figured I had to create an issue to discuss the situation that a clean LinuxGSM sof2 server does not seem to work with a clean GOG sof2 client, that's why I didn't ask anything about it. I only mentioned it as background information. 

But you're saying the question I posted, whether the 1fx. Mod was incorporated in the sof2 server as mentioned by tkrn in the start post, should also get a new issue?

If yes, I'll create the two issues in the coming days :)
